specification/maps/resource-manager/Microsoft.Maps/preview/2023-12-01-preview/maps-management.json

```diff
@@ -929,9 +923,19 @@
               "Azure-AsyncOperation": {
                 "description": "Operation Status Location URI",
                 "type": "string"
+              },
+              "Location": {
```

Comment:
>Location [](http://example.com/codeflow?start=15&length=8)

Location header isn't needed for PUT. The async model is to GET the resource and check the provisioningState value until it reaches a terminal state. Ref: [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#creating-or-updating-resources-asynchronously](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#creating-or-updating-resources-asynchronously).

Reply:
I thought the lack of `Location` in PUT response was an omission / ambiguity in the async api spec. In any case, I'd rather leave it in because it's confusing to have different monitoring semantics for PUT vs PATCH, and polling provisionState will still work.

Reply:
Removed `Location` from PUT.